include/uapi/linux/netfilter/nf_tables.h

```diff
@@ -164,6 +164,7 @@ enum nft_hook_attributes {
 enum nft_table_flags {
 	NFT_TABLE_F_DORMANT	= 0x1,
 };
+#define NFT_TABLE_F_MASK       (NFT_TABLE_F_DORMANT)
```

Comment:
I see why this is brought in ... it was introduced with this change in the upstream: `netfilter: nftables: introduce table ownership` https://github.com/torvalds/linux/commit/6001a930ce0378b62210d4f83583fc88a903d89d 

I however do not know why it was brought in during this cherry pick.

Nothing in the upstream commit touches this file, nor uses this `define`

Other than some fuzz with the `EXPORT_SYMBOL_GPL` in our code there appears to be minimal fuzz

Reply:
Just copying Red Hat TBH - breaks compile to drop it.

Reply:
Ok the first use of this `define NFT_TABLE_F_MAS` is in this change ~2 commits prior, I think we should bring this change in there
https://github.com/ctrliq/kernel-src-tree/pull/2/commits/95b04bb85e9cc2255d9b9d3e2c268f57e38e6a59

Reply:
Moved to the correct patch / commit

Reply:
Thank You for the update.